src/support/compare.ts

```diff
@@ -101,14 +148,32 @@ export interface SpliceRecord {
 }
 
 /**
+ * A record that describes how to instantiate a new object via a constructor function
+ * @param Ctor The constructor function
+ * @param args Any arguments to be passed the constructor function
```

Comment:
Any arguments to be passed the constructor function -> Any arguments to be passed to the constructor function ?